index.html

```diff
@@ -1703,12 +1713,18 @@
 			'mapnum': {},
 			'order': priority_match([modules['I']['order'], modules['II']['order'], modules['III']['order']]),
 			'money': priority_match([modules['I']['money'], modules['II']['money'], modules['III']['money']]),
+			// create tokens
+			getNameTokens: getNameTokens,
```

Comment:
Where does it get called, other than in the line immediately blow this? It's certainly not necessary for the following line.